### PR TITLE
Add most severe accidents table urban

### PR DESCRIPTION
**Pull request by ziv17**

### CODE REVIEW

File: anyway/widgets/all_locations_widgets/most_severe_accidents_table_widget.py [hunk 1]
```diff
@@ -1,174 +1,169 @@
...
+            resolution=resolution,
+        )
+        # Add casualties
+        for accident in accidents:
+            accident["type"] = AccidentType(accident["accident_type"]).get_label()
+            dt = accident["accident_timestamp"].to_pydatetime()
+            accident["date"] = dt.strftime("%d/%m/%y")
+            accident["hour"] = dt.strftime("%H:%M")
+            accident["killed_count"] = get_casualties_count_in_accident(
+                accident["id"], accident["provider_code"], 1, accident["accident_year"]
```
Comment (BarVolunteering) [inline anyway/widgets/all_locations_widgets/most_severe_accidents_table_widget.py:126]: Maybe use const/enum instated of 1 ?

File: anyway/widgets/all_locations_widgets/most_severe_accidents_table_widget.py [hunk 2]
```diff
@@ -1,174 +1,169 @@
...
+            )
+            accident["severe_injured_count"] = get_casualties_count_in_accident(
+                accident["id"], accident["provider_code"], 2, accident["accident_year"]
```
Comment (BarVolunteering) [inline anyway/widgets/all_locations_widgets/most_severe_accidents_table_widget.py:129]: Maybe use const/enum?

File: anyway/widgets/all_locations_widgets/most_severe_accidents_table_widget.py [hunk 3]
```diff
@@ -1,174 +1,169 @@
...
+            )
+            accident["light_injured_count"] = get_casualties_count_in_accident(
+                accident["id"], accident["provider_code"], 3, accident["accident_year"]
```
Comment (BarVolunteering) [inline anyway/widgets/all_locations_widgets/most_severe_accidents_table_widget.py:132]: Maybe use const/enum?

File: anyway/widgets/all_locations_widgets/most_severe_accidents_table_widget.py [hunk 4]
```diff
@@ -1,174 +1,169 @@
...
+            ).get_label()
+        return accidents
+
+    @staticmethod
+    def localize_items(request_params: RequestParams, items: Dict) -> Dict:
+        if request_params.lang != "en":
+            for item in items["data"]["items"]:
+                try:
+                    item["accident_severity"] = _(item["accident_severity"])
+                    item["type"] = _(item["type"])
```
Comment (BarVolunteering) [inline anyway/widgets/all_locations_widgets/most_severe_accidents_table_widget.py:155]: Can there be an `item["type"]` with out `item["accident_severity"]`?

File: translations/he/LC_MESSAGES/messages.po [hunk 5]
```diff
@@ -194,6 +194,35 @@ msgstr "תאונות"
...
+#: anyway/widgets/all_locations_widgets/most_severe_accidents_table_widget.py:154
+msgid "accident_severity"
+msgstr "חומרת תאונה"
+
+#: anyway/widgets/all_locations_widgets/most_severe_accidents_table_widget.py:155
+msgid "type"
+msgstr "סוג"
+
+#: anyway/widgets/all_locations_widgets/most_severe_accidents_table_widget.py:169
+#, fuzzy
```
Comment (BarVolunteering) [inline translations/he/LC_MESSAGES/messages.po:218]: The fuzzy flag prevent the use of this specific translation - we might need to recompile after removing the fuzzy flag, is this intentional?

https://codehunter.cc/a/flask/flask-babel-translations-de-lc-messages-messages-po-is-marked-as-fuzzy-skipping

**Review by BarVolunteering — COMMENTED** [review]
Looks good!
A few questions

File: anyway/widgets/all_locations_widgets/most_severe_accidents_table_widget.py (hunk 4 above)
Comment (ziv17) [reply anyway/widgets/all_locations_widgets/most_severe_accidents_table_widget.py:164]: No, both are keys in the `dict` elements in the list `items["data"]["items"]`. I do not understand, what is the issue?

File: anyway/widgets/all_locations_widgets/most_severe_accidents_table_widget.py (hunk 2 above)
Comment (ziv17) [reply anyway/widgets/all_locations_widgets/most_severe_accidents_table_widget.py]: Good catch

File: anyway/widgets/all_locations_widgets/most_severe_accidents_table_widget.py (hunk 4 above)
Comment (BarVolunteering) [reply anyway/widgets/all_locations_widgets/most_severe_accidents_table_widget.py:164]: If there is a case where only `item["type"]` can be present then we can get an KeyError on `item["accident_severity"]` and not translating `item["type"]`.
But it's not relevant

**Review by BarVolunteering — APPROVED** [review]
Looks good!

**Pull request merged**